interfaces/repo.go

```diff
@@ -747,35 +747,30 @@ func (r *Repository) RemoveSnap(snapName string) error {
 
 // DisconnectSnap disconnects all the connections to and from a given snap.
 //
-// The return value is a list of names that were affected.
-func (r *Repository) DisconnectSnap(snapName string) ([]string, error) {
+// The return value is a set of snaps that were affected.
+func (r *Repository) DisconnectSnap(snapName string) (map[*snap.Info]bool, error) {
```

Comment:
Sorry if it's glaringly obvious to you, but where exactly do you use the info, other than to get the name?